src/view/writer.js

```diff
@@ -271,6 +273,36 @@ export function mergeContainers( position ) {
 }
 
 /**
+ * Breaks given `range` on a set of {@link module:engine/view/range~Range ranges}, that each are contained within a
+ * {@link module:engine/view/containerelement~ContainerElement container element}. After `range` is broken, it's "pieces" can
+ * be used by {@link module:engine/view/writer~writer} (which expects that passed ranges are contained within on container element).
```

Comment:
Well, it applies to all methods in writer, so first one is correct, but the other is not.